Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Private/QualisysLiveLinkRetargetAsset.cpp

```diff
@@ -131,6 +131,11 @@ void UQualisysLiveLinkRetargetAsset::BuildPoseFromAnimationData(float DeltaTime,
 {
     const auto& SourceBoneNames = InSkeletonData->GetBoneNames();
 
+    // Make sure we're dealing with a QTM animation skeleton
+    const auto SanityCheckIndex = SourceBoneNames.Find("Hips");
+    if (SanityCheckIndex < 0)
+        return;
```

Comment:
Is this correct? This would include the sports markerset as well. And, do we want this check here at all?

Reply:
If the skeleton being streamed is a custom skeleton then it will likely not have the "Hips" joint or the toe joints used in the scaling calculation.  This was causing an indexing error in that section of the code which assumes it's there.  It felt easiest to put the check in at the top of the section and just skip the whole thing if it wasn't an animation (or sports) skeleton.

Reply:
I guess when using custom skeletons there is no need to use the retarget asset?

Reply:
That is correct.